test/storage_snapper_configure_snapper_test.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env rspec
+
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+
+Yast.import "StorageSnapper"
+Yast.import "Storage"
+
+
+describe "StorageSnapper#configure_snapper?" do
+
+
+  it "configures snapper" do
+
+    data = {
+      "device" => "/dev/sda1",
+      "mount" => "/",
+      "used_fs" => :btrfs,
+      "userdata" => { "/" => "snapshots" }
+    }
+
+    Yast::Storage.stub(:GetEntryForMountpoint).with("/").once.and_return(data)
+
+    expect(Yast::StorageSnapper.configure_snapper?).to eq true
```

Comment:
Frankly this rspec syntax is nuts. There's a good way to assign a function to a variable in ruby - but no, rspec does something different. Why not ```subject = lambda { ... }```?

And these pseudo sentences where you either have to use either a space, dot or underscore is just as bad. What's the justification for ```to eq true``` and ``` to be_true```?